protocol/avdl/tlf_keys.avdl

```diff
@@ -0,0 +1,26 @@
+@namespace("keybase.1")
+
+protocol tlfKeys {
+  import idl "crypto.avdl";
+
+  @typedef("string")
+  record CanonicalTlfName {}
+
+  record TLFCryptKeys {
+    // canonical name of the TLF
+    CanonicalTlfName CanonicalName;
+
+    // the KeyGen of the first element stored in SecretKeys
+    int FirstValidKeyGen;
+
+    // crypt keys of all generations for this TLF, in order, starting from
+    // FirstValidKeyGen
+    array<Bytes32> CryptKeys;
+  }
+
+  /**
+   getTLFCryptKeys returns TLF crypt keys from all generations.
+   */
+  TLFCryptKeys getTLFCryptKeys(string tlfName);
```

Comment:
One is from service to kbfs (TlfKeys) and the other from cli to service (Tlf).

I was thinking the one from cli to service (Tlf) might include other Tlf related stuff in the future, but not sure if there's ever gonna be. Should I just use the same interface?